mousetrap.js

```diff
@@ -1018,4 +1029,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
It's not really meant to actually do anything on the server side. You would still need to `require` it though if you are working on something that renders in both a server and browser environment (such as React). Most module bundlers don't like conditional requires. So it basically ends up `require`d on the server side, but you don't actually use the library for anything that runs server side (in React, there are lifecycle methods that only run on the client side, so that's when you would create your mousetrap bindings).